app/controllers/questions/consent_controller.rb

```diff
@@ -43,7 +43,7 @@ def after_update_success
         current_intake.client.tax_returns.replace(tax_returns)
         tax_returns.map { |tr| tr.advance_to(:intake_in_progress) }
 
-        sign_in current_intake.client
+        sign_in_and_clear_intake_session current_intake.client
```

Comment:
Overriding `sign_in` or warden hook is nicer because it changes fewer files. Or maybe less nice because it's spooky action at a distance.